templates/content.mustache

```diff
@@ -9,15 +9,9 @@
     		{{> title.mustache}}
     	{{/unless}}
 
-      <!--
-      <ul class="on-this-page fixed open">
-        <li><a href="#">Monkeys</a></li>
-        <li class="active"><a href="#">Fish</a></li>
-        <li><a href="#">Cats</a></li>
-        <li><a href="#">Dogs</a></li>
+      <ul class="on-this-page">
```

Comment:
In this case, the only reason I left the ul in is because theveryone template runs when the site is generated and I build the TOC dynamically in the client.  So I couldn't make it conditionall without redoing all of that.

I can remove the ul if there are no h2s when the TOC code runs.